codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/ActionServiceSourceGenerator.scala

```diff
@@ -137,13 +137,13 @@ object ActionServiceSourceGenerator {
       }
     }
 
-    s"""|$unmanagedComment
-        |
-        |package $packageName;
+    s"""|package $packageName;
         |
         |$imports
         |
-        |/** An action. */
+        |/**
+        | * An action.$unmanagedComment
```

Comment:
I think that comment is more a developer description than something that belongs in api doc section. Could we have it as a separate comment (as it was before, or with `//` style) right after the imports but not inside the api doc?

Reply:
works for me, updated